File: mario.c
#include <cs50.h>
#include <stdio.h>

void print_row(int bricks);

int main(void)
{
    //prompt for input
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1);

    //print pyramid
    for (int i = 0; i < height; i++)
    {
        for (int s = height - i; s > 0; s--)
        {
            printf(" ");
        }
        print_row(i+1);
    }
}
void print_row(int bricks)
{
    for (int i = 0; i < bricks; i++)
    {
        printf("#");
    }
    printf("\n");
}

File: mario.py
from cs50 import get_int

while True:
    height = get_int("Height : ")
    if (type(height) == int and int(height) >= 1 and int(height) <= 8):
        break

for i in range(1, height + 1):
    print(" " * (height - i) + "#" * i)
